### ISSUE TITLE
Containers do not restart on reboot

### ISSUE THREAD

**Issue by prasadtalasila** [INTO-CPS-Association/DTaaS#256]
Traefik v2.10 does not restart on OS reboot. ~~This needs to be checked again and if needed traefik version need to be downgraded.~~

**Comment by prasadtalasila** [comment]
The problem is not with the containers. The `--restart always` option should be given to all the containers. Only then will they restart. The traefik-gateway launch with restart:

```sh
docker run -d \
 --name "traefik-gateway" \
 --network=host -v "$PWD/traefik.yml:/etc/traefik/traefik.yml" \
 -v "$PWD/auth:/etc/traefik/auth" \
 -v "$PWD/dynamic:/etc/traefik/dynamic" \
 -v /var/run/docker.sock:/var/run/docker.sock \
 --restart always \
 traefik:v2.10
```

All the `docker run` commands must be checked for this problem.

@nichlaes , please fix this bug before starting issue #301. Thanks.

**Comment by prasadtalasila** [comment]
PR #304 fixes the problem.
